plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/OpenShiftPreferencePage.java

```diff
@@ -128,6 +128,9 @@ private boolean validateLocation(String location) {
 			cliLocationEditor.setErrorMessage(NLS.bind("{0} does not have execute permissions.", file));
 			return false;
 		}
+		if (!new OCBinaryValidator(location).isCompatibleForPublishing()) {
+	        setMessage("Your Openshift 3 client executable is pre-1.1.1 and may cause rsync problems", WARNING);
```

Comment:
afaics you don't know if it is pre-1.1.1 match could also hit post-1.1.1. 

Suggest to have warning be like: "OpenShift client is not 1.1.1 and may cause rsync problems if it is pre-1.1.1".